### PR TITLE
web: remove path from analytics

### PR DESCRIPTION
**Pull request by jazzdan**

### CODE REVIEW

File: web/src/HUD.tsx [hunk 1]
```diff
@@ -89,7 +89,11 @@ class HUD extends Component<HudProps, HudState> {
   componentWillMount() {
     incr("ui.web.init", { ua: window.navigator.userAgent })
     this.unlisten = this.history.listen((location, _) => {
-      let tags = { path: location.pathname }
+      let isLog =
```
Comment (nicks) [inline web/src/HUD.tsx]: can you add of mappings from pathname -> tags? this feels very brittle

File: web/src/HUD.tsx [hunk 2]
```diff
@@ -89,7 +89,11 @@ class HUD extends Component<HudProps, HudState> {
...
+        location.pathname.startsWith("/r/") &&
+        !location.pathname.endsWith("/preview")
+      let isPreview = location.pathname.endsWith("/preview")
+      let tags = { isLog: isLog.toString(), isPreview: isPreview.toString() }
```
Comment (nicks) [inline web/src/HUD.tsx]: would it make sense to add a digest if it's available, like we do for other fields? https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/digest

Comment (jazzdan) [reply web/src/HUD.tsx]: In the future yeah but for now I didn't care about the path names for their own sake I just wanted to be able to tell what kind of paths they were.

File: web/src/HUD.tsx (hunk 1 above)
Comment (jazzdan) [reply web/src/HUD.tsx]: Yeah let me merge this in now but I'll make myself a ticket to do that tomorrow.

**Pull request merged**